apps/extension/src/core/domains/mnemonics/migrations/index.ts

```diff
@@ -36,7 +36,13 @@ const mnemonicAccountTypes: AccountType[] = [
 
 const getMnemonicHash = async (cipher: string, password: string) => {
   const { val: mnemonic, err } = await decryptMnemonic(cipher, password)
-  if (err) throw new Error(mnemonic)
+  if (err) {
+    if (mnemonic === MnemonicErrors.IncorrectPassword) {
+      // set a value in localStorage storage
+      window.localStorage.setItem("mnemonicMigrationError", "true")
```

Comment:
This is where we initially detect that something has gone wrong, and flag it in localstorage